CONTRIBUTING.md

```diff
@@ -50,6 +52,17 @@ There are several linter commands available to help ensure the plugin follows th
 
 To test a WordPress plugin, you need to have WordPress itself installed. If you already have a WordPress environment setup, use the above Create Block Theme build as a standard WordPress plugin by putting the `create-block-theme` directory in your wp-content/plugins/ directory.
 
+### Repository Management
+
+Members of the [Block Themers GitHub team](https://github.com/orgs/WordPress/teams/block-themers) have write access to the repository. The team is made up of contributors who have:
+
+- Demonstrated a commitment to improving how block themes are built in the editor
+- Made 2-3 meaningful contributions to the plugin (similar to the [Gutenberg team requirements](https://developer.wordpress.org/block-editor/contributors/repository-management/#teams))
+
+If you meet this criterion and would like to be added to the Block Themers team, feel free to ask in the [#core-editor](https://make.wordpress.org/chat/) or [#core-themes](https://wordpress.slack.com/archives/C02RP4VMP) Slack channels.
```

Comment:
```suggestion
If you meet this criteria and would like to be added to the Block Themers team, feel free to ask in the [#core-editor](https://make.wordpress.org/chat/) or [#core-themes](https://wordpress.slack.com/archives/C02RP4VMP) Slack channels.
```

Nitpick: I believe criterion is singular and criteria, plural. Since we have 2 conditions, I think the plural fits better 😄

Reply:
Oh yes, you're right, thank you!